cd/before-deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+if [ "$TRAVIS_BRANCH" = 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
```

Comment:
I copy-pasted, but this sort of travis environment variable inspection is common.  Travis takes the 1 config per project approach (vs separate PR and deploy configs in jenkins), so it is sure to expose [nice stuff](https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables).